tests/sentry/models/test_environment.py

```diff
@@ -22,3 +30,9 @@ def test_simple(self):
         )
 
         assert env2.id == env.id
+
+        with self.assertNumQueries(0):
```

Comment:
I was going to add this to the second `get_or_create` call but turns out that always tries to do an `INSERT` on `sentry_environmentproject` which wasn't what I was expecting